packages/calypso-e2e/src/lib/components/cookie-banner-component.ts

```diff
@@ -1,28 +1,36 @@
-import { Page } from 'playwright';
+import { Page, Locator } from 'playwright';
 
 const selectors = {
-	acceptCookie: 'input[value="Close and accept"]',
+	acceptCookie: '.a8c-cookie-banner__ok-button',
 };
 
 /**
  * Represents the cookie banner shown on pages when not logged in.
  */
 export class CookieBannerComponent {
-	private page: Page;
+	private page: Page | Locator;
```

Comment:
Not sure if adding something so specific to the editor such as `editorWindow` would be a good idea for this cookie banner component which is supposed to work in other contexts than the editor.

The reason why I added this is because there some circumstances where the cookie banner is not reachable within the page due to an iframe or similar, so we need to provide a more specific container for the cookie to be found.

That means that sometimes, the cookie would be within the `Page` instance and sometimes would be within the `Locator` instance.

In order to unblock this PR, I can add a second property called `container` (so it supports more scenarios than the editor) and then update the method to try locating first the cookie banner within the page and fallback to the given container if not found, but I honestly consider that adds unnecessary complexity.

Reply:
> Not sure if adding something so specific to the editor such as editorWindow would be a good idea for this cookie banner component which is supposed to work in other contexts than the editor.

AFAIK the CookieBannerComponent as it is implement only appears on the editor, thus the concern is unnecessary.

> I honestly consider that adds unnecessary complexity.

I would argue otherwise - the APIs between these components are designed to be very similar, and keeping the meaning behind terms such as "page" is important. Making the "page" act as a "locator" or "page" that only differs based on the calling method is not keeping this contract.

Reply:
Fair enough! I'll implement these changes as soon as the test site switches back to a block theme.

Reply:
Solved in https://github.com/Automattic/wp-calypso/pull/71333/commits/b43c6dcf7ae7c74411b09c29f29ab454a25785bc